src/main/java/school/lemon/changerequest/java/introduction/pr1/Bob.java

```diff
@@ -1,10 +1,30 @@
 package school.lemon.changerequest.java.introduction.pr1;
 
 
+import java.util.Scanner;
+
 public class Bob {
 
     public static String hey(String phrase) {
-        return "";
+        if (phrase == null || phrase.trim().length() == 0) {
+            return "Fine";
+        }
+        if (phrase.matches("^[A-Z\\p{Lu}]+[\\d\\W]*$")) {
+                return "Chill out!";
```

Comment:
There is actually no need for Regexp here, but if you decided to use it - analyze all test examples to find out what is common situation of yelling.
Try to solve this task from other point of view: can you check if given phrase is 100% not yell ?
Anyway you need to determine condition when given phrase is yell.